fineract-provider/src/test/java/org/apache/fineract/infrastructure/core/LiquibaseStepDefinitions.java

```diff
@@ -65,6 +65,13 @@ public LiquibaseStepDefinitions() {
         });
         Given("Liquibase is enabled with a default tenant", () -> {
             initializeLiquibase(true);
+            initializeWriteInstance();
```

Comment:
Let's create a new StepDefinitions class for the isntance types, like InstanceTypeStepDefinitions that only stores the related functionalities.

Technically what you implemented works but it's definitely not at the right place.